packages/@storefront/core/CHANGELOG.md

```diff
@@ -6,9 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased] [minor]
 ### Added
-- SF-1125: Added option for parsing collection/area through URL on details pages.
+- SF-1125: Added option for parsing `collection` and `area` through URL on details pages.
```

Comment:
Sorry I didn't catch this initially, but you should also mention that this is implemented in the default parser, and also outline what they need to include in their return value to be able to use this.